resources/istio/charts/grafana/dashboards/galley-dashboard.json

```diff
@@ -296,14 +296,14 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(container_cpu_usage_seconds_total{container_name=~\"galley\", pod_name=~\"istio-galley-.*\"}[1m]))",
+          "expr": "sum(rate(container_cpu_usage_seconds_total{job=\"kubernetes-cadvisor\",container_name=~\"galley\", pod_name=~\"istio-galley-.*\"}[1m]))",
```

Comment:
The CPU panel is empty now. Indeed I cannot find a job with that name... please double-check what the goal of that change is

Reply:
reverted

Reply:
sill there